src/modules/manageLikes.js

```diff
@@ -0,0 +1,30 @@
+const itemLike = async (itemId) => {
+  await fetch(
+    `https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/vne0sKL7BtYmXIhUvGK9/likes/`,
+    {
+      method: "POST",
+      headers: {
+        "Content-type": "application/json; charset=UTF-8",
+      },
+      body: JSON.stringify({
+        item_id: itemId,
+      }),
+    }
+  );
+};
+const likeCount = async (id) => {
+  const likeEntries = await fetch(
+    `https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/vne0sKL7BtYmXIhUvGK9/likes`
+  );
+
+  await likeEntries.json().then((likeEntry) => {
+    const likeCounter = document.getElementById(`like-count${id}`);
+    likeEntry.forEach((like) => {
+      if (id === like.item_id) {
+        likeCounter.innerHTML = `${like.likes}`;
+      }
+    });
+  });
+};
+
+export { itemLike, likeCount };
```

Comment:
- [ ] Sometimes when I click on like it doesn't respond until I click multiple times and the number goes from 0 to 4 so I suggest fixing this bug so that When I click one click should the button responds display the right number of likes in order to complete the requirements

Reply:
Can you please suggest a way to fix this?

Reply:
Thankyou for the suggestion, Now like are updated immediately